packages/jaeger-ui-components/src/TraceTimelineViewer/SpanDetail/AccordianText.test.js

```diff
@@ -33,24 +33,26 @@ describe('<AccordianText>', () => {
   };
 
   beforeEach(() => {
-    wrapper = mount(<AccordianText {...props} />);
+    const r = render(<AccordianText {...props} />);
```

Comment:
i'd just move the `render` function out of the `beforeEach` and into each test. then you don't need `rerender` at all.